### PR TITLE
add optional_arg ignore_warning

### PR DESCRIPTION
**Pull request by sincerywaing**
### Description of Issue

add ignore_warning as optional_arg to avoid unnecessary crash.

issue #179

### CODE REVIEW

File: napalm_junos/junos.py [hunk 1]
```diff
@@ -204,8 +206,13 @@ def _load_candidate(self, filename, config, overwrite):
 
             if fmt == "xml":
                 configuration = etree.XML(configuration)
-
-            self.device.cu.load(configuration, format=fmt, overwrite=overwrite)
+            # add ignore_warning optional arg
+            if self.ignore_warning:
```
Comment (dbarrosop) [inline napalm_junos/junos.py]: Ok, the PR is good but the if..else is not really necessary. You can just write it as:

```
    self.device.cu.load(configuration, format=fmt, overwrite=overwrite,
                        ignore_warning=self.ignore_warning)
```

**Comment by dbarrosop** [comment]
Great! it's pretty much done, just a minor change. Could you also update the documentation, please? Just a brief note here:

https://github.com/napalm-automation/napalm/blob/develop/docs/support/index.rst#list-of-supported-optional-arguments

File: napalm_junos/junos.py (hunk 1 above)
Comment (sincerywaing) [reply napalm_junos/junos.py]: lol make much sense! thanks David!

File: napalm_junos/junos.py [hunk 2]
```diff
@@ -204,8 +206,9 @@ def _load_candidate(self, filename, config, overwrite):
 
             if fmt == "xml":
                 configuration = etree.XML(configuration)
-
-            self.device.cu.load(configuration, format=fmt, overwrite=overwrite)
+            # add ignore_warning optional arg
```
Comment (dbarrosop) [inline napalm_junos/junos.py]: Remove the comment please

**Comment by dbarrosop** [comment]
Great, all green! I will merge when we have settled napalm-automation/napalm/pull/389 :)

Thanks for the PRs!

**Comment by sincerywaing** [comment]
Thanks David!!! Really excited to contribute!

**Pull request merged**